core/field_angle.js

```diff
@@ -121,18 +117,23 @@ Blockly.FieldAngle.WRAP = 360;
 Blockly.FieldAngle.RADIUS = Blockly.FieldAngle.HALF - 1;
 
 /**
- * Adds degree symbol and recalculates width.
- * Saves the computed width in a property.
+ * Creates the block UI for this field.
+ * @package
+ */
+Blockly.FieldAngle.prototype.init = function() {
+  Blockly.FieldAngle.superClass_.init.call(this);
+  var symbol = Blockly.utils.createSvgElement('tspan', {}, null);
```

Comment:
Add a comment: // Add the degree symbol to the left of the number, even in RTL (issue #2380)

Reply:
Fixed